unittest/gunit/CMakeLists.txt

```diff
@@ -315,10 +315,6 @@ IF(MY_COMPILER_IS_GNU AND NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12)
   ADD_COMPILE_FLAGS(regexp_engine-t.cc COMPILE_FLAGS "-Wno-restrict")
 ENDIF()
 
-IF(MY_COMPILER_IS_GNU AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 8)
-  ADD_COMPILE_FLAGS(hypergraph_optimizer-t.cc connect_joins-t.cc COMPILE_FLAGS "-Wno-unused-variable")
-ENDIF()
-
```

Comment:
Is this related to encryption?

Reply:
No, it's unrelated, but looks like it didn't cause any build issues for me and I didn't notice it. I'll fix it.